docs/queries/about.mdx

```diff
@@ -347,52 +347,6 @@ screen.getByText('text', {
 })
 ```
 
-## Debugging
```

Comment:
I thought about it as well, but when you go through the docs, you don't expect to find anything about debugging here, especially since there is a separate section dedicated to it elsewhere. That's why it took me a long time to discover those two functions.

However, it wouldn't take much effort to add a link to the debugging section here.

We can mention both functions:

:::tip
You can make use of [`screen.debug` and `screen. logTestingPlaygroundURL`](link-here) to troubleshoot queries.
:::

Reply:
Yea, you're right. Let's leave it like this.